### DISCUSSION TITLE
Modifying label for the color picker

### DISCUSSION THREAD

**Discussion by HashimHaider** [GrapesJS/grapesjs#4088] [Q&A]
Hi guys,

For the [built-in 'color' trait](https://grapesjs.com/docs/modules/Traits.html#built-in-trait-types), I'd like to modify the rendered label's text. Right now, the name of the 'property' property is used as the label. 
Is there perhaps a 'title' or 'label' property that I could set which would then set the color picker's label?

An example of what I want to do:

`COLOR_PICKER = {
    property: 'color',
    type: 'color',
    label: 'Select a color' // This is ignored. The value inside 'property' is rendered as a label.
}
        
styleManager.addSector('test', {
    name: 'Test',
    open: true,
    properties: [COLOR_PICKER]
}, {at: 0});`

TIA!

**Comment by artf** [comment]
The `label` property (in StyleManager, not traits from TraitManager) was introduced in the latest version.
